webapp/config/routes/api.yaml

```diff
@@ -0,0 +1,14 @@
+# Add the API controllers twice, both with the /api/v4 as well as the /api prefix
+domjudge_api_v4:
+    type: annotation
+    resource: ../../src/Controller/API/
+    prefix: /api/v4
+    name_prefix: v4_
```

Comment:
I moved this to a seperate file. The `name_prefix` is needed because it needs to be unique per loaded routing set, otherwise /api would not work.